curator-framework/src/main/java/org/apache/curator/framework/api/transaction/TransactionCreateBuilder.java

```diff
@@ -18,17 +18,17 @@
  */
 package org.apache.curator.framework.api.transaction;
 
-import org.apache.curator.framework.api.ACLCreateModePathAndBytesable;
-import org.apache.curator.framework.api.ACLPathAndBytesable;
-import org.apache.curator.framework.api.Compressible;
-import org.apache.curator.framework.api.CreateModable;
-import org.apache.curator.framework.api.PathAndBytesable;
-
-public interface TransactionCreateBuilder<T> extends
-    PathAndBytesable<T>,
-    CreateModable<ACLPathAndBytesable<T>>,
-    ACLPathAndBytesable<T>,
-    ACLCreateModePathAndBytesable<T>,
-    Compressible<ACLCreateModePathAndBytesable<T>>
+public interface TransactionCreateBuilder<T> extends TransactionCreateBuilder2<T>
```

Comment:
Yeah, it looks cool, I'll have to have a play when I get some time. Unfortunately, I'm not doing anything ZK related at work at the moment, so it will have to wait for now.